lib/VeresOneDidDoc.js

```diff
@@ -384,99 +385,102 @@ class VeresOneDidDoc {
           }
         }
       ],
+      patch,
+      sequence,
       target: this.id,
-      sequence: this.meta.sequence,
-      patch
     };
   }
 
   /**
    * Composes and returns a service id for a service name.
    *
-   * @param serviceName {string}
+   * @param {string} serviceName
```

Comment:
Why switch the order? Both are valid JSDocs syntax.

The problem with the new format is - Putting the variable name _after_ the type declaration often results in awkward duplication, like below, `@param {string} serviceId Service id`.
I strongly advocate for variable name - then type.

Reply:
does it work either way?  This is the way the jsdoc docs show, and is also how we're doing it elsewhere.

Reply:
It works either way, yeah.

Reply:
to address your concern about readability, I favor the hyphen between the variable name and the description, soo...
```
`@param {string} serviceId - the service ID.`
```

Reply:
A valid approach, sure. I've tended not to use it cause it takes up _two whole extra characters_ for the docstring, which are in premium demand, due to 80 char limit :)